includes/Core/Modules/Module_Sharing_Settings.php

```diff
@@ -205,4 +200,27 @@ public function get_shared_roles( $slug ) {
 		return array();
 	}
 
+	/**
+	 * Merges two arrays recursively to a specific depth.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $array1 First array.
+	 * @param array $array2 Second array.
+	 * @param int   $depth Optional. Depth to merge to. Default is 1.
+	 *
+	 * @return array Merged array.
+	 */
+	private function array_merge_deep( $array1, $array2, $depth = 1 ) {
```

Comment:
Might be nice to extract this to a utility method, this can wait though...

Reply:
I remember having a similar discussion previously and colocating it in the consuming class was the consensus unless the util is being used in multiple place. So I followed that. We should definitely relocate it to a shared util class like the `BC_Functions` one if/when this gets used elsewhere.

Reply:
Thanks for pointing that out @kuasha420, it's a fair comment!

Also - apologies, as I didn't spot that you had worked on this too. I saw Evan had created the PR so gave him the shout out on my approval message, but hadn't spotted your commits so neglected to tag you too. It's a small thing, but I am glad to have now amended! :)